assets/css/style.css

```diff
@@ -416,6 +416,45 @@ footer ul li a:hover {
   text-decoration: underline;
 }
 
+#status {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 12px;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+#status>div {
+  padding: 2px 6px 1px 6px;
+  border-radius: 3px;
+}
+
+#status.red #support, #status.red #installation {
+  background-color: var(--pink);
+  color: white;
+}
+
+#status.red>a {
+  background-color: var(--pink);
+  color: white;
+  padding: 2px 6px 1px 6px;
```

Comment:
Would it be more accessible to use em and rem than px?
(I won't block on this - can be done in a separate CR if prudent)

Reply:
Thanks for pointing this out, use em/rem for the whole css file need more efforts.